File: Even Swap [31 - March 2023]/Brute Force/Recursive.java
int [] lexicographicallyLargest(int [] arr, int n) {
		
		boolean swap=false;
		
		for(int i=0;i<n-1;i++)
	    {
	         if((arr[i]+arr[i+1])%2==0 && arr[i]<arr[i+1])
	         {
	             int temp=arr[i];
	             arr[i]=arr[i+1];
	             arr[i+1]=temp;
			 
			 swap=true;
	         }
	    }
		
		if(!swap) return arr;
		
		
		    return lexicographicallyLargest(arr,n);
		
	}

File: Even Swap [31 - March 2023]/Brute Force/Recursive.cpp
//		>> Recursive.cpp << 

vector<int> lexicographicallyLargest(vector<int>& a, int n) {
        bool swap=false;
        
        for(int i=0;i<n-1;i++) {
            if((a[i]+a[i+1])%2==0 && a[i]<a[i+1]) {
                int temp=a[i];
                a[i]=a[i+1];
                a[i+1]=temp;
                swap=true;
            }
        }
        if(!swap) return a;
        return lexicographicallyLargest(a,n);
    }
